formfactory/models.py

```diff
@@ -180,6 +182,11 @@ class Form(BaseFormModel):
 in all cases. Since it incurs a performance penalty you may wish to disable \
 it.""")
     )
+    paragraph = SimpleMDEField(
```

Comment:
I'm not sure about this. I think we should introduce LabelFields or ParagraphFields because then we can position the text using our existing formfactory techniques.

Reply:
I agree with that approach. Much better.